src/main/java/com/hivemq/util/ThreadFactoryUtil.java

```diff
@@ -18,12 +18,11 @@
 
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
 import com.hivemq.extension.sdk.api.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.ThreadFactory;
 
-/**
- * @author Florian Limpöck
```

Comment:
We're keeping author tags in the CE, right?

Reply:
From what I know from Silvio, we keep authors only for APIs